solidity/test/rewards/TestECDSARewardsDistributorEscrow.js

```diff
@@ -0,0 +1,143 @@
+const {accounts, contract, web3} = require("@openzeppelin/test-environment")
+const {createSnapshot, restoreSnapshot} = require("../helpers/snapshot")
+const {expectRevert} = require("@openzeppelin/test-helpers")
+const {expect} = require("chai")
+
+const KeepToken = contract.fromArtifact("KeepToken")
+const PhasedEscrow = contract.fromArtifact("PhasedEscrow")
+const ECDSARewardsEscrowBeneficiary = contract.fromArtifact(
+  "ECDSARewardsEscrowBeneficiary"
+)
+const ECDSARewardsDistributor = contract.fromArtifact("ECDSARewardsDistributor")
+const ECDSARewardsDistributorEscrow = contract.fromArtifact(
+  "ECDSARewardsDistributorEscrow"
+)
+
+describe("ECDSARewardsDistributorEscrow", () => {
+  const owner = accounts[0]
+  const thirdParty = accounts[1]
+
+  let token
+  let rewardsDistributor
+  let escrow
+
+  before(async () => {
+    token = await KeepToken.new({from: owner})
```

Comment:
We assume tokens are minted on `KeepToken` deployment. Can we add a check if `token.balanceOf(owner)` is greater than zero or even equal to a fixed value? 
We could also replace `allTokens` in tests with a fixed value instead of `token.balanceOf(owner)`. If the balance is zero some of the tests can give false positive results.

Reply:
Done in https://github.com/keep-network/keep-ecdsa/pull/631/commits/cef62684f746804ca80a2ea566df7c085c54932d. I used the fixed supply for ECDSA staker rewards.